src/sentry/backup/exports.py

```diff
@@ -177,6 +181,31 @@ def export_in_organization_scope(
     )
 
 
+def export_in_config_scope(src, *, indent: int = 2, printer=click.echo):
+    """
+    Perform an export in the `Config` scope, meaning that only models directly related to the global
+    configuration and administration of an entire Sentry instance will be exported.
+    """
+
+    # Import here to prevent circular module resolutions.
+    from sentry.models.user import User
+    from sentry.models.userpermission import UserPermission
+    from sentry.models.userrole import UserRoleUser
+
+    # Pick out all users with admin privileges.
+    admin_user_pks: set[int] = set()
+    admin_user_pks.update(UserPermission.objects.values_list("user_id", flat=True))
```

Comment:
do we need to filter models by permission/role?

Reply:
Discussed offline, confusion was over the fact that **all** `UserRole`/`UserPermission` are for admins, not just some of them.